services-js/commissions-search/src/FacetList.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import "./FacetList.css";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class FacetList extends Component {
+  render() {
+    const GET_AREAS = gql`
+      {
+        policyTypes {
+          id
+          name
+        }
+      }
+    `;
+
+    function Area(props) {
+      return (
+        <label className="cb">
+          <input
+            id="checkbox-area"
+            name={props.id}
+            type="checkbox"
+            value={props.areaValue}
```

Comment:
`value` should just be `props.id`, and use `checked` to see if it's enabled or not.